app/person-escort-record/index.js

```diff
@@ -1,31 +1,31 @@
 // NPM dependencies
-const router = require('express').Router()
+const router = require('express').Router({ mergeParams: true })
```

Comment:
Gah! I can't believe I didn't know this option existed. I think this means some rubbish code in the move updates can be removed

Reply:
Haha, yeah I remember realising it was a big game-changer when I discovered it ;)